pkg/api/types.go

```diff
@@ -14,6 +15,49 @@ const (
 	StatusError        OpenSCAPStatus = "Error"
 )
 
+// ScanResult represents the compacted result of all scans performed on the image
+type ScanResult struct {
+	// ImageName is a full pull spec of the input image
+	ImageName string `json:"imageName"`
+	// ImageIUD is a SHA256 identifier of the scanned image
+	ImageID string `json:"imageID"`
+	// Results contains compacted results of various scans performed on the image.
+	// Empty results means no problems were found with the given image.
+	Results []Result `json:"results,omitempty"`
```

Comment:
I don't think following kube API patterns makes too much sense here (already have been into this discussion with Clayton)... Also I don't think there is any example/reference in Kube for "reports".